circle.yml

```diff
@@ -14,6 +14,13 @@ dependencies:
   cache_directories:
     - "~/.cabal"
 
+  override:
+    - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
+    - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install
+    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - cd /home/ubuntu/git-town && bundle install
```

Comment:
You can use the `post` or `pre` in order for `bundle install` to still be automatic and happen before or after all these commands.